### ISSUE TITLE
Package Control doesn't uninstall TabsExtra cleanly

### ISSUE THREAD

**Issue by hgranholm** [facelessuser/TabsExtra#27]
I still have TabsExtra options in my tab right-click menu, even though the package is uninstalled. Have restarted Sublime and even the whole OS X.

<img width="304" alt="screen shot 2015-08-12 at 10 33 31" src="https://cloud.githubusercontent.com/assets/208696/9218737/9c36a7f8-40dd-11e5-930b-a1d49050f175.png">

**Comment by facelessuser** [comment]
Since it looks like you didn't do the default override menu, you should be able to just delete the `TabsExtra` folder in your `User` folder.  There is no uninstall hook in Package Control to clean up such things.

**Comment by facelessuser** [comment]
I will try and update the documentation with this info.

**Comment by facelessuser** [comment]
I take it back, Package Control did add a `remove` event hook.  I should probably use that in the future.

**Comment by hgranholm** [comment]
Yeah, that fixes it, thank you! (Still a good idea to support Package Control all the way, I'd say :-)
